contrib/runners/orquesta_runner/tests/unit/test_output_schema.py

```diff
@@ -197,9 +197,12 @@ def test_fail_incorrect_output_schema(self):
         self.assertEqual(ac_ex_db.status, ac_const.LIVEACTION_STATUS_FAILED)
 
         expected_result = {
-            "error": "Additional properties are not allowed",
-            "message": "Error validating output. See error output for more details.",
+            "stdout": "",
+            "stderr": "Additional properties are not allowed",
+            "exit_code": 1,
+            "result": "None",
```

Comment:
The structure of the expected result is different for each runner. What you have patched here is only valid for shell runner. I doubt this error will be caught for workflow execution. 1) Can you review each action runner (i.e. local shell, python, orquesta, etc.)? 2) Can you write a unit test for each action runner?

Reply:
@m4dcoder As per recent research it is looking like this problem can be solved with minor changes in UI. Let me try with changes in UI if they don't work I will surely work on reviewing and unit tests on each runner.